packages/discovery-provider/plugins/pedalboard/apps/solana-relay/src/routes/relay/relay.ts

```diff
@@ -69,6 +81,84 @@ const forwardTransaction = async (logger: Logger, transaction: string) => {
   )
 }
 
+const sendTransactionWithRetries = async ({
+  transaction,
+  commitment,
+  confirmationStrategy,
+  sendOptions,
+  logger
+}: {
+  transaction: VersionedTransaction
+  commitment: Commitment
+  confirmationStrategy: TransactionConfirmationStrategy
+  sendOptions?: SendOptions
+  logger: Logger
+}) => {
+  const serializedTx = transaction.serialize()
+  const connection = connections[0]
+
+  const start = Date.now()
+
+  // Try to send once w/ await before retrying
```

Comment:
for the future - i know this is parity but i actually wonder if we should just start w shotgun.

Reply:
was on the fence about it, so lets do it!